kythe/cxx/common/kzip_writer.h

```diff
@@ -39,6 +39,12 @@ class KzipWriter : public IndexWriterInterface {
   /// \param encoding Encoding to use for compilation units.
   static StatusOr<IndexWriter> Create(
       absl::string_view path, KzipEncoding encoding = DefaultEncoding());
+  /// \brief Constructs a Kzip IndexWriter which will create and write to
+  /// \param path Path to the file to create. Must not currently exist.
+  /// \param encoding Encoding to use for compilation units.
+  /// Same as `Create` except it returns a pointer.
+  static StatusOr<std::unique_ptr<IndexWriter>> CreateUnique(
+      absl::string_view path, KzipEncoding encoding = DefaultEncoding());
```

Comment:
Thanks for the suggestion. I added a new function because I got a bit lost in the excavation of `IndexWriter` into `unique_ptr`.  Removed and ironed things up.